libs/plugin/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ set(plugin_headers
   hpx/plugin/virtual_constructor.hpp
   hpx/plugin/detail/dll_dlopen.hpp
   hpx/plugin/detail/dll_dlopen.hpp
```

Comment:
This header is listed twice. Should we add a check for that?

Reply:
I feel like it is visually easily detectable but I can add a check for that if you wish, what are the consequences in visual studio if one of the file is listed twice ?

Reply:
No consequences, so no need to add a check. I was merely asking whether this is worth doing.

Reply:
Yep I'll do that but probably later in another PR